ipn/ipnlocal/c2n.go

```diff
@@ -265,6 +274,10 @@ func handleC2NPprof(b *LocalBackend, w http.ResponseWriter, r *http.Request) {
 }
 
 func handleC2NSSHUsernames(b *LocalBackend, w http.ResponseWriter, r *http.Request) {
+	if !buildfeatures.HasSSH {
+		http.Error(w, feature.ErrUnavailable.Error(), http.StatusNotImplemented)
```

Comment:
Will this (separately) require any special handling on the control plane? I imagine we might at least want to count those for visibility into build feature usage.

Reply:
😭